daps-xslt/asciidoc/postprocess.xsl

```diff
@@ -170,4 +170,12 @@
     </xsl:if>
   </xsl:template>
 
+  <xsl:template match="d:formalpara[d:para/d:screen]">
+    <!-- Issue #569 -->
```

Comment:
This comment does not explain the issue and to understand it, I need to open a Web page. I would omit that. We  already have a listing of the issues addressed by this stylesheet at the top, so it makes more sense to explain the issue/solution there.

Reply:
Done.